website/client/src/components/achievements/chooseClass.vue

```diff
@@ -121,44 +121,48 @@
   }
 
   .class-explanation {
-    font-size: 16px;
-    margin: 1.5em auto;
+    font-size: 1rem;
```

Comment:
Are we changing to use `rem` for font-size?

Reply:
We aren't. Updated to `em`

Reply:
I would prefer we use `rem` over `em`.

Reply:
oh I see! If `rem` is good let's do it!
Definitely not `em`, where did you get that idea 😵‍💫

Reply:
Back to `rem` we go! 🫠